terraform/azure/variables.tf

```diff
@@ -212,7 +212,7 @@ variable "hana_data_disks_configuration" {
   type = map(any)
   default = {
     disks_type       = "Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS"
-    disks_size       = "128,128,128,128,128,128,128"
+    disks_size       = "32,32,16,16,32,64,32"
```

Comment:
Maybe we can add a comment in the code to say something like

```
This default is aligned to default for hana_vm_size
```

We can also update comment in line 197 to talk about `Standard_E4s_v3` and not `Standard_M32ls`

Reply:
Each value now is using a different number: does it make sense to document why and which disk has which size? Where we should do it? Here in this file or in the readme?

Reply:
I think those disks sizes are not finalized.  I'll collect some data about cost saving with this commit.  Then there shall be a follow up ticket to optimize the disk sizes further.  Then we can add definitive documentation about disk sizes.